conf/application.conf

```diff
@@ -242,13 +230,13 @@ microservice {
 scheduling {
   online-testing {
     send-phase1-invitation-job {
-      enabled = false
```

Comment:
We can't check these in turned on. Will break ATs

Reply:
Arrgh these were not supposed to be checked in